handlers.go

```diff
@@ -527,8 +473,9 @@ func updateRec(db dbType,
 	if err != nil {
 		return NORET, err
 	}
-	ivals := strListToInterfaces(dbrec.Values)
-	result, err := stmt.Exec(ivals...)
+	defer stmt.Close()	// nolint
+	// ivals := strListToInterfaces(dbrec.Values)
+	result, err := stmt.Exec(dbrec.Values...)
```

Comment:
I would suggest use `db.Begin()`, `tx.Commit()` and `tx.RollBack()` to make it a transaction, in order to have atomicity.